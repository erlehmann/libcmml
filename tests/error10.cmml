<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!DOCTYPE cmml SYSTEM "cmml.dtd">

<cmml lang="pl">

<stream basetime="npt:0">
  <import contenttype="video/mpeg" src="fish.mpg" start="npt:0"/>
</stream>

<head lang="en">
  <title>Types of fish</title>
  <meta name="Producer" content="Joe Ordinary"/>
  <meta name="DC.Author" content="Joe's friend"/>
</head>

<head lang="de">
  <title>Fischarten</title>
</head>

<clip lang="de" id="intro" start="npt:0">
  <a href="http://www.blah.au/fish.html">Lesen Sie mehr &#252;ber Fische.</a>
  <desc lang="en">This is the introduction to the film Joe made about fish.</desc>
</clip>

<clip lang="en" id="goldfish" start="npt:5:5.9">
  <a href="http://www.blah.au/morefish.anx?id=goldfish">More video clips on goldfish.</a>
  <img src="http://www.blah.au/goldfish.jpg"/>
  <desc>Joe has a fishtank at home with many colourful fish. The common goldfish is one of them and Joe's favourite. Here are some fabulous pictures he has taken of them.</desc>
  <meta name="Location" content="Joe's fishtank"/>
  <meta name="Subject" content="goldfish"/>
  <meta lang="fr" name="sujet" content="poisson d'or"/>
</clip>

</cmml>
