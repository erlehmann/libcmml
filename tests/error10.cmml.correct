<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!DOCTYPE cmml SYSTEM "cmml.dtd">

<cmml lang="pl">

<stream basetime="npt:0.000">
<import contenttype="video/mpeg" src="fish.mpg" start="npt:0.000"/>
</stream>

<head lang="en">
<title>Types of fish</title>
<meta name="Producer" content="Joe Ordinary"/>
<meta name="DC.Author" content="Joe's friend"/>
</head>

