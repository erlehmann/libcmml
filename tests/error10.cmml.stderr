error10.cmml:16:0: Multiple head tags not allowed
